lib/node_modules/@stdlib/_tools/changelog/parse-commits/lib/conventional_changelog.js

```diff
@@ -0,0 +1,276 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var visitWithAncestors = require( 'unist-util-visit-parents' );
+var visit = require( 'unist-util-visit' );
+
+
+// VARIABLES //
+
+var STDLIB_ISSUE_URL = 'https://github.com/stdlib-js/stdlib/issues/';
```

Comment:
Rather than hardcode these URLs, it would be better to pass in as options to the main export (e.g., `opts.issueURL` and `opts.prURL` or `opts.issue_url` and `opts.pr_url`). This would help make this package more general.

Reply:
This has meanwhile been implemented.